benchmark_runner/common/ssh/ssh.py

```diff
@@ -35,7 +36,9 @@ def run(self, cmd: str, is_check: bool = False, background: bool = False):
                 output = subprocess.getoutput(cmd)
             return output
         except subprocess.CalledProcessError as err:
-            logger.error(f'subprocess Status : FAIL: {err.returncode} {err.output}')
-            raise SSHSubprocessError()
+            logger.info(f'subprocess Command: {cmd}')
```

Comment:
Why did you delete the logger.error ?

Reply:
I'll remove this commit, I tried to limit the flow of error messages / length of callstack to made my debugging easier, but that's personal taste